packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts

```diff
@@ -235,66 +288,100 @@ export class SummarizerNode implements IRootSummarizerNode {
 		readAndParseBlob: ReadAndParseBlob,
 		correlatedSummaryLogger: ITelemetryLogger,
 	): Promise<RefreshSummaryResult> {
-		this.defaultLogger.sendTelemetryEvent({
-			eventName: "refreshLatestSummary_start",
-			proposalHandle,
-			referenceSequenceNumber: this.referenceSequenceNumber,
-			summaryRefSeq,
-		});
+		/**
+		 * Refresh latest summary should never happen while a summary is in progress. If it does, log an error
+		 * so that we can gather data around it and take appropriate action.
+		 */
+		if (this.isTrackingInProgress()) {
```

Comment:
Is this a specific GC summarizer node thing? Also, maybe valuable to converge both the summarizerNode and the summarizerNodeWithGC together.

Reply:
isTrackingInProgress() is a generic summarizer node API.
No, we intentionally kept summarizer node and summarizerNodeWithGC separate since summarizer node is a generic base imlement that tracks summaries. You can build any number of state tracking on top of it including GC.